Makefile

```diff
@@ -12,22 +12,22 @@ medius:
 	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o bin/medius kubevirt.io/containerdisks/cmd/medius
 
 fmt:
-	go mod tidy -compat=1.19
+	go mod tidy -compat=1.21
```

Comment:
Isn't required to specify the Z version?

Reply:
Not here.